recipes/apr-util/all/conanfile.py

```diff
@@ -0,0 +1,204 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class AprUtilConan(ConanFile):
+    name = "apr-util"
+    description = "The Apache Portable Runtime (APR) provides a predictable and consistent interface to underlying platform-specific implementations"
+    license = "Apache-2.0"
+    topics = ("conan", "apr-util", "apache", "platform", "library")
+    homepage = "https://apr.apache.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False],
+        "with_nss": [True, False],
+        "with_commoncrypto": [True, False],
+        "dbm": [False, "gdbm", "ndbm", "db"],
+        "with_expat": [True, False],
+        "with_mysql": [True, False],
+        "with_postgresql": [True, False],
+        "with_sqlite3": [True, False],
+        "with_lber": [True, False],
+        "with_ldap": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_openssl": True,
+        "with_nss": False,
+        "with_commoncrypto": False,
+        "dbm": False,
+        "with_expat": True,
+        "with_mysql": True,
```

Comment:
proposed default_options probably dont align with how configure works, as discussed please set all of these defaults to 'False'.
With current recipe if someone installs the default mode it adds to requires mysql postgresql and sqlite which is a bid crazy.

Reply:
not sure if expat can be disabled though, when I tried to locally i got:
Expat cannot be disabled (at this time)

Reply:
I've added a check that will raise an error when you try to disable it.
If it can be disabled it in the future, a version check can be added.